dropwizard-configuration/src/main/java/io/dropwizard/configuration/ConfigurationMetadata.java

```diff
@@ -159,8 +159,11 @@ public void optionalProperty(BeanProperty prop) throws JsonMappingException {
                 parentProps.add(prop);
 
                 // visit the type of the property (or its defaultImpl).
-                mapper.acceptJsonFormatVisitor(
-                        defaultImpl == null ? fieldType.getRawClass() : defaultImpl, thiss);
+                try {
+                    mapper.acceptJsonFormatVisitor(defaultImpl == null ? fieldType.getRawClass() : defaultImpl, thiss);
+                } catch (NoClassDefFoundError | Exception e) {
+                    e.printStackTrace();
```

Comment:
I'm a bit conflicted about this.
On the one hand I don't want to swallow the exception quietly, on the other hand this pollutes the output when an application starts without Logback on the classpath.

@dropwizard/committers Any opinions?